packages/golden_toolkit/test/configuration_test.dart

```diff
@@ -173,6 +175,40 @@ void main() {
       );
     });
 
+    group('Shadows', () {
+      GoldenToolkit.runWithConfiguration(
+        () {
+          testGoldens(
+              'screenMatchesGolden method uses enableRealShadows from global configuration when GoldenToolkit.runWithConfiguration is outside testGoldens',
+              (tester) async {
+            await tester.pumpWidgetBuilder(WidgetWithShadows());
+            await screenMatchesGolden(
+                tester, 'enableRealShadows_honored_when_testGoldens_wrapped');
+
+            expect(debugDisableShadows, isFalse,
+                reason: 'debugDisableShadows should be false during this test');
+          });
+        },
+        config: GoldenToolkitConfiguration(enableRealShadows: true),
+      );
+
+      testGoldens(
+          'screenMatchesGolden method does not use enableRealShadows from global configuration when GoldenToolkit.runWithConfiguration is inside testGoldens',
+          (tester) async {
+        await GoldenToolkit.runWithConfiguration(
+          () async {
+            await tester.pumpWidgetBuilder(WidgetWithShadows());
+            await screenMatchesGolden(tester,
+                'enableRealShadows_ignored_when_testGoldens_not_wrapped');
```

Comment:
you'll see that `enableRealShadows_ignored_when_testGoldens_not_wrapped.png` has ugly opaque block shadows